package.json

```diff
@@ -60,7 +60,7 @@
   "scripts": {
     "lint": "populist lib/*.js lib/winston/*.js lib/winston/**/*.js",
     "test": "mocha",
-    "test:coverage": "nyc npm run test:unit",
+    "test:coverage": "nyc npm run test",
```

Comment:
@fearphage This one I disagree with. Coverage is typically not a great metric for determining what code is thoroughly & sufficiently tested because it's typically the integration aspect that will cause unexpected/undesired behavior. 

That being said, running coverage against integration tests typically inflate your coverage numbers (presuming you have true separation of integration & unit/very narrow integration tests). This is due to the possibility of the application inadvertently taking code paths that were not intentionally be tested. Thereby you can get code "covered" that has no unit/narrowed integration tests.

I'd much prefer our coverage report be on the low side to help push for unit tests to be built out, rather than letting integration tests gives us coverage because it touched a piece of functionality that we may not have explicitly been testing.

Reply:
Agreed.